README.md

```diff
@@ -27,6 +29,25 @@ Note: In general, track parameters are only defined for a subset of the challeng
 | --------- | ----------- | ---- | ------------- |
 | `record_raw_event_size` | Adds a new field `_raw_event_size` to the index which contains the size of the raw logging event in bytes. | `bool` | `False` |
 
+Note: It is recommended to store any track parameters in a json file and pass them to Rally using '--track-params=./params-file.json'. 
+
+Follow is an example of a valid paramter json file:
```

Comment:
typos: **Following** ... **parameters**

Reply:
Typos aside, one issue here is that we are stating that the following is an example of a **valid** parameters json file, but since it uses parameters specific to the `elasticlogs-continuous-index-and-query` challenge this is not very accurate.

What if we just show a minimal example that is applicable to all challenges:

``` json
{
    "shard_count": 3,
    "number_of_replicas": 1
}
```

Reply:
Yeah, I was thinking something along those same lines, but wasn't confident to write a generic one yet, so I just copied it from below, I'll adjust that.

Reply:
This should be fixed now.

Reply:
Looks good, but there is still the typo mentioned here: https://github.com/elastic/rally-eventdata-track/pull/55/files#r349474128

Reply:
This is fixed